tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceWrapperCommand.java

```diff
@@ -39,6 +39,22 @@ public ServiceWrapperCommand( IServer server )
         _server = server;
     }
 
+    public String[] getServiceWrapper() throws Exception
+    {
+
+        if( _server == null )
+        {
+            return getStaticServiceWrapper();
+        }
+        else
+        {
+            String[] wrappers = getDynamicServiceWrapper();
+            updateServiceWrapperStaticFile( wrappers );
```

Comment:
This updatestatic file should not update in the plugin install area but should update in the plugin state area instead.